Makefile

```diff
@@ -1,6 +1,29 @@
 .PHONY: all get-deps clean compile run eunit check check-eunit doc
 
-REBAR=$(shell which rebar || echo ./rebar)
+#
+REBAR2=$(shell which rebar || echo ./rebar)
+REBAR3=$(shell which rebar3)
+ifeq ($(FORCE_REBAR2),true)
+ REBAR=$(REBAR2)
+ REBAR_VSN=2
+else ifeq ($(REBAR3),)
+ REBAR=$(REBAR2)
+ REBAR_VSN=2
+else
+ REBAR=$(REBAR3)
+ REBAR_VSN=3
+endif
+ifeq ($(REBAR_VSN),2)
+ TARGET_ALL_DEPS = get-deps compile
+ TARGET_EUNIT_DEPS = compile
+ TARGET_CHECK_DEPS = dializer-2
```

Comment:
Looks like this spelling is also disparate from [here](https://github.com/erlcloud/erlcloud/pull/256/files#diff-b67911656ef5d18c4ae36cb6741b7965R85)